src/main/java/com/Studentlifr/GratitudeBoard/controller/UserController.java

```diff
@@ -0,0 +1,31 @@
+package com.Studentlifr.GratitudeBoard.controller;
+
+import com.Studentlifr.GratitudeBoard.dto.requests.MessageReqDTO;
+import com.Studentlifr.GratitudeBoard.dto.response.ResponseDTO;
+import com.Studentlifr.GratitudeBoard.services.MessageService;
+import com.Studentlifr.GratitudeBoard.services.impl.TokenServiceImpl;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.validation.Valid;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("user")
+public class UserController {
+    private final MessageService messageService;
+    private final TokenServiceImpl tokenService;
+
+    public UserController(MessageService messageService, TokenServiceImpl tokenService) {
+        this.messageService = messageService;
+        this.tokenService = tokenService;
+    }
+
+    @PostMapping("postMessage")
```

Comment:
we should not use REST verbs in the resource path,

POST  users/postMessage
POST users/messages